docs/workflows/how-to/fail-pipeline-gently.md

```diff
@@ -0,0 +1,71 @@
+# Fail pipeline without stopping it immediately
```

Comment:
the heading is a bit long, what about something like 'Fail  pipeline gracefully'